### PR TITLE
Update CONTRIBUTING.md with thanks stickers

### PR DESCRIPTION
**Pull request by Molkobain**
Hey everyone,

We just made some stickers to thank all the contributors of the iTop ecosystem, here is a PR to prepare the contributing guide. Feel free to help modeling the sentences as I'm having an hard time finding a good sentence 😅
Once this is done, we will contact all contributors from this year to get their email / postal address so we can send them to them.

Depending on how this go, we might add more kind of stickers in the near future.

Cheers ✌

*Note: Please don't tell us you want to change the stickers design as they are already printed 😁*

### CODE REVIEW

**Comment by Molkobain** [comment]
(Will I get 1 for making this PR? 🤔)

**Comment by Molkobain** [comment]
@jbostoen @Hipska @larhip @purplegrape @DudekArtur @nv35 @samsk @annProg @vbkunin @lazki feel free to comment and to send us your postal address at guillaume.lajarige (at) combodo (dot) com

**Review by Hipska — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -125,8 +125,7 @@ Our tests are located in the `test/` directory, containing a PHPUnit config file
     * ⬆️ `:arrow_up:` when upgrading dependencies
     * ⬇️ `:arrow_down:` when downgrading dependencies
     * ♻️ `:recycle:` code refactoring
-    * 💄 `:lipstick:` Updating the UI and style files.  
-
```
Comment (piRGoif) [inline CONTRIBUTING.md:129]: Can you revert this unrelated change ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -137,3 +136,15 @@ When your code is working, please:
 * create a pull request.
 
 Detailed procedure to work on fork and create PR is available [in GitHub help pages](https://help.github.com/articles/creating-a-pull-request-from-a-fork/).
+
+### 🙏 We are thankful
```
Comment (piRGoif) [inline CONTRIBUTING.md:144]: I'm not very found of this title

Maybe simply : 

```suggestion
### 🙏 Many thanks to our great contributors !!
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -137,3 +136,15 @@ When your code is working, please:
...
+
+We are thankful for all your contributions to the iTop universe, here are the 2021 stickers that will be sent to every iTop (& extensions) contributors this year. Design might change from one year to another, for the first year we wanted to try a "craft beer label" look.
```
Comment (piRGoif) [inline CONTRIBUTING.md]: My suggestion : 

```suggestion
We are thankful for all your contributions to the iTop universe ! As a thank you gift, we will send nice stickers to every iTop (& extensions) contributors ! 

Stickers' design might change from one year to another. For the first year we wanted to try a "craft beer label" look, see examples below : 
```

**Review by piRGoif — DISMISSED** [review]
Bravooooooo for this great initiative !
I can suggest some changes in the formulation though O:)

**Review by larhip — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (Hipska) [reply CONTRIBUTING.md]: "we will send nice stickers to every.." 😉

Comment (piRGoif) [reply CONTRIBUTING.md]: I've changed the suggestion, thanks !

File: CONTRIBUTING.md (hunk 2 above)
Comment (Molkobain) [reply CONTRIBUTING.md:144]: I'm not fond of the multiple "!", it's too much for me 😅
Also we already say "... thankful for all the contributions ..." in the following line so it feels redundant to me.

File: CONTRIBUTING.md (hunk 3 above)
Comment (Molkobain) [reply CONTRIBUTING.md]: This sounds ok to me if:
- We remove the space before the 2 "!" and the ":"
- Replace "send nice stickers" by "send stickers" as the "nice" feels a bit pretentious to 😁

Comment (piRGoif) [reply CONTRIBUTING.md]: Hello,

Here is the amended modification : 

```suggestion
We are thankful for all your contributions to the iTop universe! As a thank you gift, we will send stickers to every iTop (& extensions) contributors! 

Stickers' design might change from one year to another. For the first year we wanted to try a "craft beer label" look, see examples below: 
```

**Review by steffunky — APPROVED** [review]

**Comment by Molkobain** [comment]
> @jbostoen @Hipska @larhip @purplegrape @DudekArtur @nv35 @samsk @annProg @vbkunin @lazki feel free to comment and to send us your postal address at guillaume.lajarige (at) combodo (dot) com

Hello @purplegrape @DudekArtur @nv35 @samsk @annProg, mind to send us an email if you us to send them to you :)

**Comment by annProg** [comment]
@Molkobain Please just send me electronic version（may be photo or scanning copy) to  tecbbs(at)qq(dot)com. Thanks!

**Comment by Hipska** [comment]
@annProg digital original for the stickers is available in this PR or here:

[](https://github.com/Combodo/iTop/blob/develop/documentation/contributing-guide/contributing-stickers-side-by-side.png)

**Comment by Molkobain** [comment]
> @Molkobain Please just send me electronic version（may be photo or scanning copy) to tecbbs(at)qq(dot)com. Thanks!

Thanks, I just sent you the images in a better quality. Cheers!

**Comment by annProg** [comment]
@Molkobain @Hipska I have received the email, thank you very much!

**Comment by purplegrape** [comment]
@Molkobain  I just wanna a electronic version too,please send mail to purplegrape4(at)gmail(dot)com ,many thanks!

**Comment by Molkobain** [comment]
> purplegrape4(at)gmail(dot)com

Thanks you! The email has just been sent.

**Pull request merged**